nntrainer/tensor/task_executor.cpp

```diff
@@ -48,9 +48,10 @@ void TaskExecutor::cancelAll(void) {
 void TaskExecutor::clean(void) {
   for (auto it = tasks.begin(); it != tasks.end();) {
     auto running = std::get<std::atomic_bool>(it->second).load();
-    if (running == false)
-      tasks.erase(it);
+    auto cur = it;
```

Comment:
```suggestion
    if (running == false)
      it = tasks.erase(it);
    else
      it++;
```
It's a bug. please check.

Reply:
I modified it. Thank you.